app/src/main/java/com/example/myapplication/SecondFragment2.java

```diff
@@ -71,12 +76,36 @@ public View onCreateView(
         return binding.getRoot();
     }
 
+    /*Suppressed due to server having no way to get the dynamic path
+        TODO: Add matchNumber to file name once added
+    */
+    @SuppressLint("SdCardPath")
+    public void writeFile() throws IOException {
+        new File("/sdcard/Documents/ScoutingData/").mkdirs();
+
+        File dataFile = new File("/sdcard/Documents/ScoutingData/data.txt");
```

Comment:
This looks like the file will always be called data.txt - is there a way to differentiate between the matches for them? Or does the computer have to look inside the file to find the difference?

Reply:
I forgot to add the match number 💀